content/en/docs/Getting started/_index.md

```diff
@@ -5,23 +5,122 @@ weight: 100
 description: Quick Baker setup instructions
 ---
 
-Baker is a Go library and it must be included into your Go program.
+Do you think Baker suits your needs? Good! Now let's see together how to start using it.
 
-To create a new Go project using Go modules and adding Baker:
+Baker is a Go library and, to be used, it needs to be included into a Go program.  
+In this page we describe the simplest way to use Baker. At the end of it, we recommend reading the
+[Baker Core Concepts](/docs/core-concepts/) and then have a deep dive in the
+[How-to pages](/docs/how-tos/).
+
+## Add Baker...
+
+### ...to a brand-new project
+
+To create a new Go project (using Go modules) and add Baker, these are the suggested steps:
 
 ```sh
 mkdir myProject
 cd myProject
 go mod init github.com/myUser/myProject
-go get github.com/adroll/baker
+go get github.com/AdRoll/baker
 ```
 
+### ...to an existing project
+
 If you are adding Baker to a project already configured to use Go modules, just type:
 
 ```sh
 cd myProject
-go get github.com/adroll/baker
+go get github.com/AdRoll/baker
+```
+
+## Use Baker
```

Comment:
This doesn't show how to use baker but how to build it

Reply:
it's a h2 and it includes "configuration" and "run the program" as h3. Maybe we can say: "Build and run Baker"